controllers/status.go

```diff
@@ -308,6 +318,10 @@ func getComponentStatuses(hub *operatorsv1.MultiClusterHub, allHRs []*subhelmv1.
 		components["multicluster-engine-sub"] = unmanagedStatus
 	}
 
+	if !ocpConsole {
+		components["console-chart-sub"] = consoleUnavailableStatus
```

Comment:
When !ocpConsole it will overwrite the console-chart-sub status with an Unavailable status, but that does not reflect what the actual status may be. It looks like it will still deploy the console-chart sub, but just report a different status.